### PR TITLE
Translation cleanup

### PR DESCRIPTION
**Pull request by raccube**

### CODE REVIEW

**Review by pixeldesu — APPROVED** [review]

File: config/locales/views.en.yml [hunk 1]
```diff
@@ -270,7 +270,8 @@ en:
       help:
         password: "Leave this blank if you don't want to change it"
       delete:
-        action: "Delete my account"
+        action: "Permanently delete account"
+        confirm: "Are you sure? This deleting your account is IRREVERSIBLE."
```
Comment (nilsding) [inline config/locales/views.en.yml]: This deleting is the
```suggestion
        confirm: "Are you sure? Deleting your account is IRREVERSIBLE."
```

File: config/locales/views.en.yml [hunk 2]
```diff
@@ -331,14 +332,14 @@ en:
           present: "Here's your export from %{time}"
     mutes:
       index:
-        title: "Muted Words"
-        heading: "Muted words"
+        title: "Muted Terms"
+        heading: "Muted Terms"
         body: "Muting words (or longer phrases) will prevent questions containing those to appear in your inbox."
```
Comment (nilsding) [inline config/locales/views.en.yml:337]: is it words or terms now?

Comment (raccube) [reply config/locales/views.en.yml:337]: I felt that words felt weird

**Review by nilsding — APPROVED** [review]
diese gut

**Pull request merged**